query_test.go

```diff
@@ -16,6 +18,26 @@ func TestClient_Query_Exec_allTypes(t *testing.T) {
 		t.Fatalf("failed to create client: %v", err)
 	}
 
+	id := uuid.MustParse("f066f76d-5e96-4b52-8d8a-0f51387df76b")
+	timeUUID := uuid.MustParse("30821634-13ad-11eb-adc1-0242ac120002")
+	vals := []interface{}{
+		id,                       // id
+		"alpha",                  // asciivalue
+		"bravo",                  // textvalue
+		"charlie",                // varcharvalue
+		[]byte("foo"),            // blobvalue
+		true,                     // booleanvalue
+		2.2,                      // doublevalue
+		float32(3.3),             // floatvalue
+		net.ParseIP("127.0.0.1"), // inetvalue
+		1,                        // bigintvalue
+		2,                        // intvalue
+		3,                        // smallintvalue
+		5,                        // tinyintvalue
+		big.NewInt(64),           // varintvalue
```

Comment:
Not a big deal. This is probably an okay test case, but maybe this could use something that can only be represented by a `big.Int` and is outside the range of a native type.

Reply:
How's this?